packages/paste-core/components/input-box/__tests__/input-box.test.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import {matchers} from 'jest-emotion';
+import {render, screen} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {InputBox} from '../src';
+
+expect.extend(matchers);
+
+describe('HTML attributes', () => {
+  it('should set a element data attribute for InputBox', () => {
+    render(<InputBox element="foo">input-box</InputBox>);
+    expect(screen.getByText('input-box').getAttribute('data-paste-element')).toEqual('foo');
+  });
```

Comment:
I wouldn't mind a test here for in all tests cases that ensures the default element name is set, along side a custom one.